### ISSUE TITLE
DroidCamX forgets Exposure, Autofocus and Whitebalance Settings

### ISSUE THREAD

**Issue by Gambloide** [dev47apps/droidcam#151]
**Describe the bug**
Changing any of the Exposure, Autofocus or Whitebalance settings from a default value to another value is not saved.

Steps to reproduce:
1. Start a connection between DroidCamX and a droidcam receiver, e.g. droidcam-cli on Linux.
2. Change any of these settings: Exposure, Autofocus, Whitebalance
3. Stop the connection from the receiver
4. Start the connection again

Exposure, Autofocus and Whitebalance are set back to their default values.

**Expected behavior**
DroidCamX remembers Exposure, Autofocus and Whitebalance settings.
Alternatively, enable setting those from the Windows, Mac and Linux droidcam applications so they can be saved and enforced from that side.

**Additional context**
Used droidcam-cli version: `1.7.1`
Used droidcam-cli command: `droidcam-cli -size=3840x2160 adb 4747` 
Phone: Samsung Galaxz S9 Plus Edge

**Comment by Maingron** [comment]
Agree. Would be super nice if you could just leave and connect back into the device the next time without touching the settings again.
Think it should also be possible to activate the continuous Autofocus thingy from the receiver end.

**Comment by Trenar** [comment]
Another nice persistent setting to store would be the anti-flicker

**Comment by hmlendea** [comment]
Yes! The zoom level would also be very nice to remember!

**Comment by sachingopal** [comment]
Yes please. I just bought the droidcam pro and manually changing this every time is  a pain. These options should be available through the cli.

**Comment by virtualflesh** [comment]
Persistent settings would be a wonderful feature! For OBS users (streaming or otherwise), the video feed is reset if the media source is inactive. It would be excellent if, when activated, the IP cam would be in its prior state. Thank you!

**Comment by aramg** [comment]
Hi, if you are using OBS I would **highly** recommend switching to the newer DroidCam OBS app.
It includes persistent camera settings (as part of the pro upgrade), and is under much more active development. 
Since OBS Studio now supports v4l2loopback support natively, it can act as a much superior PC client for general webcaming, too.

**Comment by virtualflesh** [comment]
> Hi, if you are using OBS I would **highly** recommend switching to the newer DroidCam OBS app. It includes persistent camera settings (as part of the pro upgrade), and is under much more active development. Since OBS Studio now supports v4l2loopback natively, it can act as a much superior PC client for general webcaming, too.

DroidCam OBS is far superior. thanks for the recommendation.

**Comment by eliliam** [comment]
Bumping this! I'd love to see zoom level either A) remembered, or B) configurable via a CLI flag or something of the sort.
